### ISSUE TITLE
DOMException thrown on destroy()

### ISSUE THREAD

**Issue by someguythatcodes** [katspaugh/wavesurfer-multitrack#31]
## Bug description
I seem to be getting the following error when calling destroy() on a multitrack object:

`Uncaught (in promise) DOMException: Failed to execute 'decodeAudioData' on 'BaseAudioContext': Unable to decode audio data`

From what I can see, destroy loops through the audio array and sets src = '' for each audio. This works for all HTMLAudioElement audios, but for WebPlayerAudios (of which the placeholder is one) setting src = '' results in WebAudioPlayer fetching '' and trying to decode the result which results in the above error. 

Loving the library so far btw!

## Environment
React App

## Minimal code snippet
``` 
const multitrack = Multitrack.create(
    [
        {
            id: 0,
            url: "/example.mp3",
            startPosition: 0
        },
        {
            id: 1,
            url: "/example2.mp3",
            startPosition: 0
        }
    ],
    {
        container: document.querySelector("#multitrack"),
        minPxPerSec: 10, // zoom level
        rightButtonDrag: false, // set to true to drag with right mouse button
        cursorWidth: 2,
        cursorColor: '#D72F21',
        trackBackground: '#2D2D2D',
        trackBorderColor: '#7C7C7C',
        dragBounds: true,
        envelopeOptions: {
            lineColor: 'rgba(255, 0, 0, 0.7)',
            // lineWidth: 4,
            dragPointSize: 8,
            dragPointFill: 'rgba(255, 255, 255, 0.8)',
            dragPointStroke: 'rgba(255, 255, 255, 0.3)',
        },
    }
)


multitrack.on("canplay", () => {
    multitrack.destroy()
})
```

## Expected result
Multitrack gracefully destroys it's audios and Wavesurfers

## Obtained result
`Uncaught (in promise) DOMException: Failed to execute 'decodeAudioData' on 'BaseAudioContext': Unable to decode audio data`

**Comment by katspaugh** [comment]
Oh, good catch!

**Comment by yarikLev04** [comment]
@katspaugh have the same, make conversation https://github.com/katspaugh/wavesurfer.js/discussions/3410 and provide code

**Comment by yarikLev04** [comment]
Hi @katspaugh, pls can you fix this bug, because it provide a lot of problems with updating multitrack data, I mean when I load one data then on the same page load another it cant be updated because of this bug

**Comment by DenysPostolnyi** [comment]
Hi, @katspaugh. I have the same problem

**Comment by lunavamp** [comment]
Hello, @katspaugh, I have this problem too

**Comment by zdebskyy** [comment]
@katspaugh hey, got same issue, would be great if you look at it) Thanks

**Comment by katspaugh** [comment]
Thanks for asking all your collegues to chime in, the more the merrier. This is now fixed in 0.4.8.
